### ISSUE TITLE
TypeError: travis.repos.builds is not a function

### ISSUE THREAD

**Issue by paulirish** [davglass/git-travis#9]
Heyo! thanks for this app.

Getting this on my first run of `git travis`

```sh
❯ git travis
Fetching build status for GoogleChrome/lighthouse:master
/Users/paulirish/.config/yarn/global/node_modules/git-travis/lib/index.js:77
    travis.repos.builds({
                 ^

TypeError: travis.repos.builds is not a function
    at fetch (/Users/paulirish/.config/yarn/global/node_modules/git-travis/lib/index.js:77:18)
    at /Users/paulirish/.config/yarn/global/node_modules/git-travis/lib/index.js:277:13
    at ClientRequest.<anonymous> (/Users/paulirish/.config/yarn/global/node_modules/git-travis/lib/index.js:154:20)
    at Object.onceWrapper (events.js:293:19)
    at emitOne (events.js:96:13)
    at ClientRequest.emit (events.js:191:7)
    at HTTPParser.parserOnIncomingClient [as onIncoming] (_http_client.js:522:21)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:99:23)
    at TLSSocket.socketOnData (_http_client.js:411:20)
    at emitOne (events.js:96:13)

```

git remote data:
```
❯ git remote -v
GoogleChrome	git://github.com/GoogleChrome/lighthouse.git (fetch)
GoogleChrome	git@github.com:GoogleChrome/lighthouse.git (push)
```

Tried to debug but dont' know the travis API well enough to be useful.

**Comment by davglass** [comment]
Sorry about that, pushed a fix in `git-travis@3.1.0` (the travis-ci module updated their API under the hood and my semver range didn't catch it :(

**Comment by paulirish** [comment]
All good. Thanks for the speedy fix!
